_posts/2025-09-04-playtonik-looping.md

```diff
@@ -3,6 +3,8 @@ layout: post
 author: Tom Cartwright
 ---
 
+<iframe width="100%" src="https://www.youtube.com/embed/ByBfa3axGeE?si=slml7JWGrF8-ROK1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen style="border-radius: 15px; aspect-ratio: 1.78"></iframe>
+
 I've recently been working on a looper feature for Playtonik, our musical fidget toy. The idea is to capture a section of the inputted notes and to give the user controls to play the loop back at will.
```

Comment:
Just thinking, worth adding an App Store Link for Playtonik here?... https://apple.co/4oTOgyW